client/src/tests/relatedProductsTests.test.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 import helper from '../helper-functions/rpHelpers';
-import exampleData from '../relatedProducts/exampleData.js';
+import exampleData from '../relatedProducts/newExampleData.js';
 import { expect, jest, test, describe, beforeEach, afterEach } from '@jest/globals';
 import RelatedProducts from '../relatedProducts/RelatedProductsView/RelatedProducts.jsx';
-import {renderer} from 'react-test-renderer';
-import { product_id } from './QnA-testData';
+import renderer from 'react-test-renderer'
```

Comment:
I know I already have renderer installed, but I imagine other team members might need a heads up as to what dependancies they are going to need to install